openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -0,0 +1,711 @@
+        self._settings = settings
+
+        # Checks on the inputs!
+        # check that both states have solvent and ligand
+        for state, label in [(stateA, 'A'), (stateB, 'B')]:
+            if 'solvent' not in state.components:
+                raise ValueError(f"Missing solvent in state {label}")
+            if 'ligand' not in state.components:
+                raise ValueError(f"Missing ligand in state {label}")
+        # check that both states have same solvent
+        if not stateA.components['solvent'] == stateB.components['solvent']:
+            raise ValueError("Solvents aren't identical between states")
+        # check that the mapping refers to the two ligand components
+        if stateA.components['ligand'] != ligandmapping.mol1:
+            raise ValueError("Ligand in state A doesn't match mapping")
+        if stateB.components['ligand'] != ligandmapping.mol2:
+            raise ValueError("Ligand in state B doesn't match mapping")
+
+    @classmethod
+    def get_default_settings(cls) -> RelativeLigandTransformSettings:
+        """A dictionary of initial settings for this creating this Protocol
+
+        These settings are intended as a suitable starting point for creating
+        an instance of this protocol.  It is recommended, however that care is
+        taken to inspect and customize these before performing a Protocol.
+
+        Returns
+        -------
+        RelativeLigandTransformSettings
+          a set of default settings
+        """
+        return RelativeLigandTransformSettings(
+            system_settings=SystemSettings(
+                constraints='HBonds'
+            ),
+            topology_settings=TopologySettings(
+                forcefield={'protein': 'amber99sb.xml',
+                            'ligand': 'openff-2.0.0.offxml',
+                            'solvent': 'tip3p.xml'},
+            ),
+            alchemical_settings=AlchemicalSettings(),
+            sampler_settings=SamplerSettings(),
+            barostat_settings=BarostatSettings(),
+            integrator_settings=IntegratorSettings(),
+            simulation_settings=SimulationSettings(
+                equilibration_length=2.0 * unit.nanosecond,
+                production_length=5.0 * unit.nanosecond,
+            )
+        )
+
+    def to_dict(self) -> dict:
+        """Serialize to dict representation"""
+        return {
+            'stateA': self._stateA.to_dict(),
+            'stateB': self._stateB.to_dict(),
+            'mapping': self._mapping.to_dict(),
+            'settings': dict(self._settings),
+        }
+
+    @classmethod
+    def from_dict(cls, d: dict):
+        """Deserialize from a dict representation"""
+        return cls(
+            stateA=ChemicalState.from_dict(d['stateA']),
+            stateB=ChemicalState.from_dict(d['stateB']),
+            mapping=LigandAtomMapping.from_dict(d['mapping']),
+            settings=dict(**d['settings']),
+        )
+
+    def run(self) -> bool:
```

Comment:
For after Tuesday maybe - shouldn't we just be importing things from `openfe.run`?

Reply:
Yeah longer term we'll need a lazy import system to prevent us from slowly grinding to a halt.  I'd rather we keep deps up top so they're visible then later apply a good fix.

I think an intermediate solution could be that this module isn't imported on the base `import openfe` but only on `from openfe.setup.methods import openmm`.